### ISSUE TITLE
Yolo 9000 Dataset ?

### ISSUE THREAD

**Issue by dexception** [philipperemy/yolo-9000#27]
I haven't been able to locate any dataset that was used to train the Yolo 9000 dataset. 
Please share the details as we are looking to only classify few categories only.

**Comment by Shawn0Hsu** [comment]
@dexception Hello, Have you solved the problem? do you know how to tag  YOLO9000 with  my own dataset? looking forward to your reply

**Comment by jiajunhua** [comment]
I haven't been able to locate any dataset that was used to train the Yolo 9000 dataset.
Please share the details as we are looking to only classify few categories only.

**Comment by pradeep-wyze** [comment]
Even I couldn't find the processed dataset for YOLO9000 -- I think the authors of the paper YOLO900 never open-sourced the dataset. 

If any of you could locate it, please share it here. Thanks

**Comment by philipperemy** [comment]
Yeah, I've never seen it either. The dataset has never been published as far I know.

**Comment by pradeep-wyze** [comment]
@philipperemy any idea how to create one? Say if I want to build something like this for 500 classes. Any rference? Thanks a lot.
